react/utils/normalize.ts

```diff
@@ -67,14 +77,149 @@ export function changeImageUrlSize(
   return `${normalizedImageUrl}${queryStringSeparator}width=${width}&height=${height}&aspect=true`
 }
 
-function findAvailableProduct(item: {
-  sellers: Array<{ commertialOffer: { AvailableQuantity: number } }>
-}) {
-  return item.sellers.find(
-    ({ commertialOffer = {} }) =>
-      commertialOffer.AvailableQuantity != null &&
-      commertialOffer.AvailableQuantity > 0
+function getPriceFromSeller(seller: ProductTypes.Seller) {
+  return seller.commertialOffer.Price
+}
+
+function isAvailable({
+  commertialOffer: { AvailableQuantity },
+}: ProductTypes.Seller) {
+  return AvailableQuantity != null && AvailableQuantity > 0
+}
+
+/**
+ * @description
+ * Find the lowest/highest `Price` between sellers
+ * @param item an SKU
+ * @param condition the condition enum used for comparison functions
+ * */
+function getBestPrice(item: ProductTypes.Item, condition: PriceConditionRule) {
+  // If there's only 1 seller, avoid filtering
+  const { sellers } = item
+
+  if (sellers.length === 1) return sellers[0].commertialOffer.Price
```

Comment:
Small nitpick, but we prefer always writing `if`s in this form to make them easier to read.

```suggestion
  if (sellers.length === 1) {
    return sellers[0].commertialOffer.Price
  }
```

Reply:
Added